src/components/ui/Input.jsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+
+const Input = (props) => {
+    const {
+        handleChange,
+        value,
+        id,
+        name,
+        type,
+        isRequired = false,
+        placeholder,
+        label,
+        position = "top",
+        labelFontSize = "5xl",
+        labelFontWeight = "normal",
+        radius = "md",
+        inputWidthSize = "",
+        inputHightSize = "",
+        inputFontSize = "3xl",
+        inputFontWeight = "light",
+
+        ...rest
```

Comment:
Following props are enought: handleChange, value, id, name, type, isRequired, placeholder, label,  radius,  inputFontSize,  inputFontWeight, inputWidthSize,

Reply:
but also i need to specific labelFontSiz according to figma it differenc from one form to anthor